src/components/tracked-controls.js

```diff
@@ -21,6 +21,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
     rotationOffset: {default: 0},
+    hand: {type: 'string'},
```

Comment:
Ah OK.   I didn't quite understand what you were getting at.  Yeah, I'm OK with removing any application handling of hand data.